.github/workflows/appimage.yml

```diff
@@ -98,6 +100,65 @@ jobs:
         parent: false
         gzip: false
 
+  appimage-static-32bit:
+    name: static 32-bit for appimage
+    runs-on: buildjet-2vcpu-ubuntu-2204-arm
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event == 'pull_request_target' && github.event.pull_request.head.sha || github.ref }}
+    - uses: docker/setup-buildx-action@v2
+    - uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    # note: we run this way instead of using rdk-devenv as the job's container because of https://github.com/actions/runner/issues/2115 (js-based actions install incomaptible node on 32-bit)
```

Comment:
I'll try and talk to you tomorrow to understand this better when I can have a fresh brain. It feels like there should be a cleaner way around this, but... I suspect you felt that way too before hitting whatever bug is going on here.

Reply:
Okay, those are some annoying bugs. Eventually this will go away as we won't need appimages after moving to static releases with the agent. But in the interim, I think you found more or less the best workarounds you can.